docs/pages/access-controls/getting-started.mdx

```diff
@@ -20,35 +20,72 @@ wrap up with creating your own role.
 
 ## Step 1/3. Add local users with preset roles
 
-Teleport provides several preset roles: `editor`, `auditor` and `access`. The
-`editor` role authorizes users to modify cluster configuration, the `auditor`
-role to view audit logs, and `access` role to access cluster resources.
+Teleport provides several preset roles: `editor`, `auditor`, and `access`.
 
+- The `editor` role authorizes users to modify cluster configuration.
+- The `auditor` role authorizes users to view audit logs.
+- The `access` role authorizes users to access cluster resources.
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
```

Comment:
Users tend not to notice that we have scope switchers at the top of docs pages. We should use something besides `ScopedBlock` here, since this is information that's important not to miss. Maybe a `Details` box?